src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -270,15 +270,16 @@ private void UpdateIndicator()
                     double percent = Maximum == Minimum ? 1.0 : (Value - Minimum) / (Maximum - Minimum);
 
                     // When the Orientation changed, the indicator's Width or Height should set to double.NaN.
+                    // Indicator size calculation should consider the ProgressBar's Padding property setting
```

Comment:
Comments says about "Padding" property, but below you substract by Margin property.